Jenkinsfile

```diff
@@ -1124,22 +1124,22 @@ pipeline {
                                            fi
                                            tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
                                            ./ftest.sh "$test_tag" $tnodes''',
-                                junit_files: "src/tests/ftest/avocado/*/*/*.xml src/tests/ftest/*_results.xml",
+                                junit_files: "install/lib/daos/TESTING/ftest/avocado/*/*/*.xml install/lib/daos/TESTING/ftest/*_results.xml",
```

Comment:
Why don't we put tests in lib64 if we are placing libraries there?

Reply:
My only thought was that they are ultimately not 64-bit-only.  That said, I don't have a strong feeling one way or the other.

Ultimately these should be packaged up as a first-class python package i think.  But that can happen down the road and after I have given it more thought.

Reply:
We will keep this approach in this PR, can have further discussion on other concerns.